compiler/qsc/src/target.rs

```diff
@@ -26,6 +28,7 @@ impl From<Profile> for RuntimeCapabilityFlags {
         match value {
             Profile::Unrestricted => Self::all(),
             Profile::Base => Self::empty(),
+            Profile::Adaptive => Self::ForwardBranching,
```

Comment:
For adaptive + reset + integer computations will we have another profile?

Reply:
@cesarzc Yes, this is just a placeholder.